src/main/java/mate/academy/model/User.java

```diff
@@ -63,4 +64,23 @@ public String toString() {
                 + ", email='" + email + '\''
                 + '}';
     }
+
+    @Override
```

Comment:
You have incorrect overridden of equals & hashcode methods

https://vladmihalcea.com/the-best-way-to-implement-equals-hashcode-and-tostring-with-jpa-and-hibernate/

https://vladmihalcea.com/how-to-implement-equals-and-hashcode-using-the-jpa-entity-identifier/